test/IRGen/autorelease.sil

```diff
@@ -10,6 +10,8 @@ import Swift
 // armv7k: [[OBJC:%objc_object]] = type
 // arm64:  [[OBJC:%objc_object]] = type
 // i386:   [[OBJC:%objc_object]] = type
+// powerpc64: [[OBJC:%objc_object]] = type
+// powerpc64le: [[OBJC:%objc_object]] = type
```

Comment:
This is an ObjC-runtime-specific test.  Please add `// REQUIRES: objc_interop` instead and revert the other changes.